src/index.tsx

```diff
@@ -1,13 +1,26 @@
 import { createRoot } from 'react-dom/client';
-import { HashRouter as Router } from 'react-router-dom';
+import { HashRouter, Navigate, Route, Routes } from 'react-router-dom';
 
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { App } from './App';
+import { Home } from './pages/Home/Home';
+import { People } from './pages/People/People';
 
 createRoot(document.getElementById('root') as HTMLDivElement).render(
-  <Router>
-    <App />
-  </Router>,
+  <HashRouter>
+  <Routes>
+    <Route path="/" element={<App />}>
+      <Route index element={<Home />} />
+      <Route path="/home" element={<Navigate to="/" replace />} />
+
+      <Route path="/people" element={<People />}>
+        <Route path=":personSlug" element={<People />} />
+        </Route>
+
+        <Route path="*" element={<h1 className="title">Page not found</h1>} />
```

Comment:
The wildcard route `*` is used to display a 'Page not found' message. Ensure that this provides a user-friendly experience and consider adding navigation options back to valid routes.